content/docs/deployment/csmoperator/drivers/powermax.md

```diff
@@ -297,9 +297,35 @@ Create a secret named powermax-certs in the namespace where the CSI PowerMax dri
     ```bash
     kubectl create configmap powermax-reverseproxy-config --from-file config.yaml -n powermax
     ```
+4. Create a configmap using below sample file. Fill in the appropriate values for driver configuration. Example: X_CSI_TRANSPORT_PROTOCOL:"ISCSI" 
+   ```yaml  
+      # Copyright © 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
+      #
+      # Licensed under the Apache License, Version 2.0 (the "License");
+      # you may not use this file except in compliance with the License.
+      # You may obtain a copy of the License at
+      #      http://www.apache.org/licenses/LICENSE-2.0
+      # Unless required by applicable law or agreed to in writing, software
+      # distributed under the License is distributed on an "AS IS" BASIS,
+      # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      # See the License for the specific language governing permissions and
+      # limitations under the License.
+      # To create this configmap use: kubectl create -f powermax-array-config.yaml
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name: powermax-array-config
+        namespace: powermax
+      data:
+        powermax-array-config.yaml: |
+          X_CSI_POWERMAX_PORTGROUPS: pg1"
+          X_CSI_TRANSPORT_PROTOCOL: "ISCSI" # Defaults to auto
```

Comment:
lets keep the protocol blank as it defaults to `auto`

Reply:
Hi @harishp8889 - are we going to update the default to ""?